packages/flutter_test/lib/src/binding.dart

```diff
@@ -848,22 +848,23 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     });
 
     return new Future<Null>.microtask(() async {
+      // testBodyResult is a Future that was created in the Zone of the
+      // fakeAsync. This means that if we await it here, it will register a
+      // microtask to handle the future _in the fake async zone_. We avoid this
+      // by calling '.then' in the current zone. While flushing the microtasks
+      // of the fake-zone below, the new future will be completed and can then
+      // be used without fakeAsync.
+      final Future<Null> resultFuture = testBodyResult.then<Null>((_) {
+        /* Do nothing. */
```

Comment:
[nit] Any reason this comment isn't just `// Do nothing.`?

Reply:
No reason.
Changed it.